serializability-checker-compiler-plugin/src/test/scala/org/virtuslab/akkasaferserializer/compiler/AkkaSerializabilityCheckerCompiler.scala

```diff
@@ -38,7 +38,7 @@ object AkkaSerializabilityCheckerCompiler {
     val compiler = new Global(settings, reporter) {
       override protected def computeInternalPhases(): Unit = {
         super.computeInternalPhases()
-        for (phase <- new AkkaSerializabilityCheckerCompilerPlugin(this).components)
```

Comment:
There's still: `AkkaSerializabilityCheckerCompiler` and `AkkaSerializabilityCheckerCompilerSpec` to be renamed AFAICS